RMtools_3D/do_RMsynth_3D.py

```diff
@@ -199,14 +200,18 @@ def run_rmsynth(dataQ, dataU, freqArr_Hz, dataI=None, rmsArr=None,
         # Multiply the dirty FDF by Ifreq0 to recover the PI
         FDFcube *= Ifreq0Arr
 
+    RMSF1D_spectra, RMSF1D_phiArr, RMSF_fwhm = get_RMSF(lamSqArr=lambdaSqArr_m2, phiArr=phiArr_radm2, 
+        weightArr=None, lam0Sq_m2=None, double=True, fitRMSFreal=False, dtype="float32")
```

Comment:
I fully agree with the idea that in cases where a 1D RMSF is well-defined that it's sensible to save it so that users don't need the full cubes. That part is perfectly fine, I'm mostly just fussing about repeating the calculation. For the POSSUM 3D pipeline, we can effectively guarantee a 1D RMSF (except maybe at the survey edges?), so a 1D RMSF is a sensible product to save storage. As a side note: maybe we need to modify rmclean3d to accept a 1D RMSF as input, so that users can perform RM-CLEAN without the 3D RMSF?

I'm mainly just worried about cases where the RMSF can't be compressed down to 1D -- in those cases, I don't think we should give users any false impressions or misleading information, so it's safer not to save a 1D RMSF.